pkg/analysis_server/lib/src/services/correction/fix.dart

```diff
@@ -185,6 +186,8 @@ class DartFixKind {
       const FixKind('INSERT_SEMICOLON', 50, "Insert ';'");
   static const LINT_ADD_OVERRIDE =
       const FixKind('LINT_ADD_OVERRIDE', 50, "Add '@override' annotation");
+  static const LINT_REMOVE_INIT_TO_NULL =
+      const FixKind('LINT_REMOVE_INIT_TO_NULL', 50, "Remove initialization to null");
```

Comment:
So REMOVE_INITIALIZER or LINT_REMOVE_INITIALIZER? We tried to do just like the other 2 linter quick fixes (LINT_ADD_OVERRIDE, LINT_REMOVE_INTERPOLATION_BRACES